### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by andrewpeterwatson**
Page routing complete. 
activeNav is stored and updated by current URL on page render.

### CODE REVIEW

File: src/components/account.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      </div>
+    );
+  }
+}
+
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators({
+    updateVersion: updateVersion
+  }, dispatch)
+}
```
Comment (tlwirtz) [inline src/components/account.js]: This is super nitpicky, but i really love spaces between functions...

File: src/components/assets.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      </div>
+    );
+  }
+}
+
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators({
+    updateVersion: updateVersion
+  }, dispatch)
+}
```
Comment (tlwirtz) [inline src/components/assets.js]: And here...

File: src/reducers/index.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import { combineReducers } from 'redux';
 
 import AppVersion from './reducer_appVersion'
+import ActiveNav from './reducer_activeNav'
 
 
 const rootReducer = combineReducers({
-  appVersion: AppVersion
+  appVersion: AppVersion,
```
Comment (tlwirtz) [inline src/reducers/index.js]: why is this here? just a placeholder?

**Comment by tlwirtz** [comment]
@andrewpeterwatson  -- left a few comments, but this looks good. Go ahead and merge! Nice job.

File: src/reducers/index.js (hunk 3 above)
Comment (andrewpeterwatson) [reply src/reducers/index.js]: Well I wanted the app version in there that links to the package.json so we could have some version tracking. I'll through it in a settings page or on the login just to have access to.

File: src/components/account.js (hunk 1 above)
Comment (andrewpeterwatson) [reply src/components/account.js]: change made!

File: src/reducers/index.js (hunk 3 above)
Comment (tlwirtz) [reply src/reducers/index.js]: sounds good
